singer-io/tap-ilevel
# v.0.0.3 periodic data items

# Description of change
Re-work periodic_data_standardized and unique key + bookmarking strategy. Add periodic_data_calculated - new endpoint.

# Manual QA steps
Ran singer-discover, singer-check-tap, and target-stitch for all endpoints. Reviewed data in downstream target DWH.
 
# Risks
Low - new tap, new endpoint. Currently in Alpha testing w/ Bytecode (110918) and Felicis (156601).
 
# Rollback steps
Revert to v.0.0.2
